server.js

```diff
@@ -0,0 +1,269 @@
+if (process.env.NODE_ENV !== "production") {
+  require("dotenv").config();
+}
+
+const http = require("http");
+const express = require("express");
+const fs = require("fs");
+const child = require("child_process");
+const ipc = require("node-ipc");
+const WebSocket = require("ws");
+const app = express();
+const passport = require("passport");
+const session = require("express-session");
+const cookieParser = require("cookie-parser");
+
+// ENV VARS
+const users = process.env.USERS.split("|");
+const botLogins = process.env.BOT_LOGINS.split("|");
+const botPasswords = process.env.BOT_PASSWORDS.split("|");
+const sessionSecret = process.env.SESSION_SECRET;
+
+const initializePassport = require("./passport-config");
+initializePassport(passport, users);
+
+ipc.config.id = "parent";
+ipc.config.retry = 1500;
+ipc.config.silent = true;
+
+const port = 3000;
+const host = "127.0.0.1";
```

Comment:
this should be `0.0.0.0` instead of `127.0.0.1` - will bind to all interfaces (and work better for docker)